customhooksreact/src/components/molecules/UserForm/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import InputField from '../../atoms/InputField/index';
+
+interface UserFormProps {
+  inputProps: {
+    value: string;
```

Comment:
why object inside object

Reply:
Using objects inside objects as props groups related properties together, improving code organization and readability, also simplifies prop management and forwarding to child components.